cmd/otel-allocator/watcher/promOperator.go

```diff
@@ -0,0 +1,155 @@
+package watcher
+
+import (
+	"fmt"
+
+	"github.com/go-kit/log"
+	"github.com/go-logr/logr"
+	allocatorconfig "github.com/otel-allocator/config"
```

Comment:
is this import from this repository? Then it should be moved to a 3rd group. Why doesn't it have the full path?

Reply:
Yes, this is referencing the local module. 

Why this module name has been chosen, I can not say.

Reply:
I think this import path is incorrect and should be changed.  Probably best to change it in a separate PR, though.